### DISCUSSION TITLE
My Favorite Cafes (uses store locator)

### DISCUSSION THREAD

**Discussion by ghost** [jakobzhao/geog495#71] [Final Project]
I had a goal since the first week in this class to make a "cafe finder", well in actuality it's just an application that stores information for my favorite cafes. I made this because saving locations on google maps becomes very messy since you cannot sort, or at least create unique icons, for certain categories (like Cafes for example). Essentially google maps only has two or three icons, making it hard to sort by something as specific as "cafe".

https://2spooky4tony.github.io/geog495_finalproject/index.html

The website is finished, but I will be adding more cafes over time as this is a project I want to use in my daily life.
Also, apologies for the late posting, I thought I submitted this already..

![Screenshot 2021-12-11 190028](https://user-images.githubusercontent.com/72472828/145698564-a83ca8e2-c4b1-433e-811c-a9d946509519.png)

**Comment by TheAdonalsium** [comment]
I really like how this is a program designed to be used in your everyday life, I've made a few of those myself. with some more work, this could be incorporated into a blog if you enjoy that, or maybe even including a short review of each place.

**Comment by alexanderlyu** [comment]
I like the functionality of the application and how it seems to be easily updated. This definitely seems like something that could be adapted for other purposes beyond cafes as well. I think one thing that could be done better is to have a base map that pops with some more color, maybe with some browns to align thematically with the cafes.

**Comment by ebar23** [comment]
I like the approach you've taken to personalize the app towards yourself. I think that you bring up a very good point about the difficulty in personalizing your google map preferences. While this app is geared more towards you and maybe you might not like too much overwhelming colors, I think that for the purpose of the overall grade on this assignment, it would be in your best interest to add a better color scheme. Your project is great and can be very useful for your day to day life, why not make it a bit more color rich :) Good job so far!

**Comment by lucretiachen99** [comment]
I think this is a great idea and very useful for students especially because a lot of us typically don't get a long lunch break so having a map with nearby cafes definitely helps save time. One thing to improve on would be to add a search bar when you add more locations to help speed up the search for students who want to find a cafe even faster. Another thing would be to make the sidebar slightly smaller so that we can see more of the map. Overall very nice color scheme, simple and concise. Great job!

**Comment by dldl1901s** [comment]
I like how you expressed your personal experience on the map. The overall design is also very neat and intuitive. However, it would be better to match the concept color. I hope the color of the pop-up window and the color of the legend are the same.

**Comment by emilyphantastic** [comment]
This application is extremely cute. I love your choice in colors and how it shows cafe's outside of the US as well. When I clicked on the one in Japan, it felt like I was traveling (lol). 
Some suggesting I have is making the area font smaller or less bold, or make the coffee cup icon bigger. I suggest this because when I first opened your map, I didn't notice the coffee cups right away and tried to click on the area names like "Queen Anne" and Belltown" since it is more bold and noticeable compared to the coffee cups icons.

**Comment by FaisalTF** [comment]
I really like the idea of a personal objective app, as well as the coffee icons you added to represent the points. I can see something like this starting where it slowly expands into personalized spots around the map kind of like a GTA map. Constructively looking at what could improve this as whole, I think the aesthetics could mesh together a bit more nicely (maybe a combination more with a bit more harmony than blue/brown/light).
